tests/commands/test_bump_command.py

```diff
@@ -831,3 +831,168 @@ def test_bump_manual_version_disallows_prerelease_offset(mocker):
         "--prerelease-offset cannot be combined with MANUAL_VERSION"
     )
     assert expected_error_message in str(excinfo.value)
+
+
+def test_bump_command_prelease_version_type_via_cli(
+    tmp_commitizen_project, mocker: MockFixture
+):
+    # PRERELEASE
+    tmp_version_file = tmp_commitizen_project.join("__version__.py")
+    tmp_version_file.write("0.1.0")
+    tmp_commitizen_cfg_file = tmp_commitizen_project.join("pyproject.toml")
+    tmp_version_file_string = str(tmp_version_file).replace("\\", "/")
+    tmp_commitizen_cfg_file.write(
+        f"{tmp_commitizen_cfg_file.read()}\n"
+        f'version_files = ["{tmp_version_file_string}"]'
+    )
+
+    create_file_and_commit("feat: new user interface")
+
+    testargs = [
+        "cz",
+        "bump",
+        "--prerelease",
+        "alpha",
+        "--yes",
+        "--version-type",
+        "semver",
+    ]
+    mocker.patch.object(sys, "argv", testargs)
+    cli.main()
+
+    tag_exists = git.tag_exist("0.2.0-a0")
+    assert tag_exists is True
+
+    with open(tmp_version_file, "r") as f:
```

Comment:
nitpick: do you think we could try the following?

```python
for version_file in (tmp_version_file, tmp_commitizen_cfg_file):
    with open(version_file, "r") as f:
        assert "0.2.0-a0" in f.read()
```

Reply:
If you agree with this, maybe we could try this technique in other tests in this PR as well